content/react/en/addons.md

```diff
@@ -8,7 +8,7 @@ commit: "dac373a"
 # Addons
 
 Storybook boasts a robust system of [addons](https://storybook.js.org/addons/introduction/) with which you can enhance the developer experience for
-everybody in your team. If you've been following along with this tutorial linearly, we have referenced multiple addons so far, and you will have already implemented one in the [Testing chapter](/test).
+everybody in your team. If you've been following along with this tutorial linearly, we have referenced multiple addons so far, and you will have already implemented one in the [Testing chapter](https://www.learnstorybook.com/react/en/test/).
```

Comment:
This probably shouldn't have the domain in the URL: `/react/en/test/` would solve the problem and make the site portable?

Reply:
@shilman Thanks for the feedback. It does make the site more portable but it won't be clickable if we read from Github though. Anyway, reading docs in Github might not be important here so I will update the PR shortly.

Reply:
The above is updated.

Reply:
Good point about github! Makes me think it would be better still to update the links to `../test`? Though it still doesn't solve the Github problem. Thanks for your patience on this.

Side note: @domyen @tmeasday maybe you should consider `addons/index.md` etc. for this reason?